src/Engine/ProtoCore/FFI/FFIExecutionManager.cs

```diff
@@ -32,11 +32,27 @@ public static FFIExecutionManager Instance
         public Assembly LoadAssembly(string name)
         {
             System.Diagnostics.Debug.Write("Trying to load assembly: " + name);
+            Assembly asm = null;
             if (System.IO.File.Exists(name))
+            {asm = Assembly.LoadFrom(name);
+                //try
```

Comment:
meant to be commented out?

Reply:
I'm still trying to figure out what's the best place to catch the exception. Thanks for the initial review though.